custom/icds_reports/templates/icds_reports/icds_app/awc-reports.html

```diff
@@ -39,8 +39,11 @@ <h2 class="center">{$ $ctrl.data.map.title $}</h2>
                     <leaflet lf-center="$ctrl.center" layers="$ctrl.layers" markers="$ctrl.markers" height="600px" width="100%"></leaflet>
                 </div>
                 <div class="row no-margin black">
-                    <h2 class="center">PSE Photo Submissions</h2>
-                    <div uib-carousel active="active" interval="0" style="height: 300px;" >
+                    <h2 class="text-center">PSE Photo Submissions</h2>
+                    <div class="alert alert-info text-center col-xs-12 col-md-offset-2 col-md-8 title" ng-show="$ctrl.isOlderThan90Days()">
+                        PSE photos older than 90 days are not available on the ICDS CAS Dashboard
```

Comment:
What happens if I select current month and still there is not photo. Would it still show this message. ?

Reply:
Yeah, missed that case.
Instead of checking whether the response has any photos, I'll check whether the first day of the requested month is older than 90 days.